lua/mapvote/server/modules/db.lua

```diff
@@ -0,0 +1,35 @@
+local DB = {}
+MapVote.DB = DB
+function DB.CreateTable()
+	sql.Query( "CREATE TABLE IF NOT EXISTS mapvote_played_maps ( map TEXT UNIQUE, play_count INTEGER NOT NULL DEFAULT 0, last_played INTEGER )" )
+end
+
+function DB.MapPlayed(map)
+    if sql.Query( string.format("INSERT OR IGNORE INTO mapvote_played_maps (map) VALUES(%s)", sql.SQLStr(map)) ) == false then
+    	print("MapVote SQLError: ", sql.LastError())
+	end
+    if sql.Query( string.format( "UPDATE mapvote_played_maps SET play_count = play_count+1, last_played=strftime('%%s') WHERE map = %s",  sql.SQLStr(map)  ) ) == false then
+    	print("MapVote SQLError: ", sql.LastError())
+	end
+end
+
+function DB.GetRecentMaps(limit)
+    if type(limit) ~= "number" then
+        error("DB.GetRecetMaps: Limit was not a number")
+    end
+    local data = sql.Query("SELECT * FROM mapvote_played_maps ORDER BY last_played DESC LIMIT " .. limit)
+    if data == false then
+        print("MapVote SQLError: ", sql.LastError())
+    end
+    return data
+end
+
+function DB.GetAllMaps()
+    local data = sql.Query("SELECT * FROM mapvote_played_maps")
```

Comment:
```suggestion
    local data = sql.Query( "SELECT * FROM mapvote_played_maps" )
```

Also I notice we only use two fields on this query when we use it:
```suggestion
    local data = sql.Query( "SELECT map, play_count FROM mapvote_played_maps" )
```